pkg/agent/route/route_linux.go

```diff
@@ -633,7 +633,8 @@ func (c *Client) AddRoutes(podCIDR *net.IPNet, nodeName string, nodeIP, nodeGwIP
 		// Set the peerNodeIP as next hop.
 		route.Gw = nodeIP
 	} else {
-		// NoEncap traffic to Node on the same subnet. It is handled by host default route.
+		// NetworkPolicyOnly mode or NoEncap traffic to a Node in a different subnet.
+		// Routing should be handled by host default routes.
```

Comment:
is "default routes" confusing here? don't we really mean "a route which is already present on the host" (as opposed to installed by Antrea)?

Reply:
Sure.